var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -69,15 +94,13 @@ class Qt(Package):
     # https://github.com/spack/spack/issues/1517
     patch('qt5-pcre.patch', when='@5:')
 
-    patch('qt4-corewlan-new-osx.patch', when='@4')
-    patch('qt4-pcre-include-conflict.patch', when='@4')
-    patch('qt4-el-capitan.patch', when='@4')
-
-    # Allow Qt's configure script to build the webkit option with more
-    # recent versions of gcc.
-    # https://github.com/spack/spack/issues/9205
-    # https://github.com/spack/spack/issues/9209
-    patch('qt4-gcc-and-webkit.patch', when='@4')
+    patch('qt4-pcre-include-conflict.patch', when='@4.8.6')
+    patch('qt4-tools.patch', when='@4+tools')
+    if not MAC_VERSION:
+        patch('qt4-gcc-and-webkit.patch', when='@4.8.6')
```

Comment:
Could you also move the above comments to here vs. removing them?

Reply:
@sethrj do you think you would have a chance to do this in the next few days?

Reply:
Yes, sorry I had a deadline and had to put the MR on hold. I'll get it done asap.

Reply:
Yikes, when I said few days I truly didn't mean few months. I did restore the comments.